src/components/Admin/index.jsx

```diff
@@ -273,12 +308,27 @@ class Admin extends React.Component {
                           }
                         </div>
                         <div className="col-12 col-md-6 text-md-right">
-                          <DownloadCsvButton
-                            id={tableMetadata.csvButtonId}
-                            fetchMethod={tableMetadata.csvFetchMethod}
-                            disabled={this.shouldDisableCsvButton(actionSlug)}
-                            buttonLabel={`Download ${actionSlug ? 'current' : 'full'} report (CSV)`}
-                          />
+                        <div className="row">
+                          <div className="col-sm-12 col-md-7 pr-0">
+                            <SearchBar
```

Comment:
```
{this.isSearchShow ? <SearchBar
                              inputLabel="Search by email:"
                              onSearch={query => updateUrl({
                                search: query,
                                page: 1,
                              })}
                              onClear={() => updateUrl({ search: undefined })}
                              value={searchQuery}
                            />:''}

```

Reply:
we are not using it yet?

Reply:
We are using it. I think I forgot to copy in the fork repository.